Changelog.md

```diff
@@ -656,1880 +721,1880 @@
 1. Set simulation time to Rendering
     * [Pull request #1415](https://github.com/gazebosim/gz-sim/pull/1415)
 
-### Gazebo Sim 6.9.0 (2022-04-14)
+### Ignition Gazebo 6.9.0 (2022-04-14)
```

Comment:
these gazebo -> ignition changes should be reverted from the changelog

Reply:
[ffe2da0](https://github.com/gazebosim/gz-sim/pull/1794/commits/ffe2da087fb41925a0fab13ee063774c1c93cae0)